docs/resources/azure_resource_groups.md.erb

```diff
@@ -13,6 +13,20 @@ common command on a group of resources.
 
 <br />
 
+## Availability
+
+This resource is available in the `inspec-azure` [resource pack](https://www.inspec.io/docs/reference/glossary/#resource-pack).  To use it, add the following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
```

Comment:
Does it make any different if we point people to the repo or the archive file off of that repo? E.g. `git: https://github.com/inspec/inspec-azure/archive/master.tar.gz`

Reply:
I'm not sure.  We may be able to use the http/tarball approach and relax the need to have git installed.

Reply:
I tested with the tarball and it was working with that instead of the git URI.